lms/djangoapps/django_comment_client/permissions.py

```diff
@@ -5,34 +5,30 @@
 import logging
 from types import NoneType
 from django.core import cache
+
+from request_cache.middleware import RequestCache
 from lms.lib.comment_client import Thread
 from opaque_keys.edx.keys import CourseKey
 
+from django_comment_common.models import all_permissions_for_user_in_course
+
 CACHE = cache.get_cache('default')
```

Comment:
Apparently not. They're used in lms/djangoapps/django_comment_client/forum/tests.py